common/helpers/campCollaborationInitials.js

```diff
@@ -5,5 +5,7 @@ import initials from './initials.js'
  * Returns two characters to display for a camp collaboration based on its user
  */
 export default function (campCollaboration) {
-  return initials(campCollaborationDisplayName(campCollaboration, null, false))
+  return campCollaboration?.abbreviation
```

Comment:
Could you add a test for this new behaviour in our existing `__tests__/campCollaborationInitials.spec.js`?